boards/riscv32/hifive1/board.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2017 Jean-Paul Etienne <fractalclone@gmail.com>
+ * Copyright (c) 2017 Palmer Dabbelt <palmer@dabbelt.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_BOARD_H
+#define __INC_BOARD_H
+
+#include <soc.h>
+
+/*
+ * UART clock configurations
+ *
+ * Define them here so that it can be replaced by global variables
+ * on other boards where the uart clock is determined dynamically
+ * following the PLL configuration
+ */
+#define uart_fe310_port_0_clk_freq    16000000
+#define uart_fe310_port_1_clk_freq    16000000
+
+/* LEDS configuration */
+#define LED0_GPIO_PORT        "gpio0"
```

Comment:
Looks like a handful of the examples use at least LED0_GPIO_PORT.  I'll dig the right numbers out of the SDK.

Thanks!